Dockerfile

```diff
@@ -1,49 +1,12 @@
-FROM postgres:14.2-alpine AS base
+FROM quay.io/ukhomeofficedigital/hocs-base-image
 
-ENV USER hocs
-ENV USER_ID 1000
-ENV GROUP hocs
-ENV NAME hocs-toolbox
-ENV PGDATA /app/data
-ENV AWS_CLI_VERSION 1.16.207
+USER 0
 
-WORKDIR /app
+RUN apk add --no-cache postgresql-client aws-cli ca-certificates curl
```

Comment:
Looks like you've removed bash, which would be nice to have on interactive shell.

Reply:
do we want this on the base image?

Reply:
I think not because this is a util